streamlit-python/README.md

```diff
@@ -45,10 +45,12 @@ This directory contains code organization by functionality, such as one distinct
    |               |-python
    |                      |-add_test.py
    |
-   |module-ui
-   |        |-src
-   |            |-ui.py
-   |            |-environment.yml
+   |-module-ui
+   |          |-src
+   |              |-ui.py
+   |              |-environment.yml
+   |          |-test
+   |              |-test_ui.py
```

Comment:
Slightly misaligned (vertical pipe character should land on same column as last char in previous line)